src/guiguts/application.py

```diff
@@ -234,8 +260,12 @@ def init_statusbar(self, statusbar):
         statusbar.add("prev img", text="<", width=1)
         statusbar.add_binding("prev img", "<ButtonRelease-1>", self.file.prev_page)
 
-        statusbar.add("see img", text="See Img", width=7)
+        statusbar.add("see img", text="See Img", width=9)
         statusbar.add_binding("see img", "<ButtonRelease-1>", self.load_image)
+        statusbar.add_binding(
+            "see img", "<ButtonRelease-3>", self.file.choose_image_dir
+        )
+        statusbar.add_binding("see img", "<Double-Button-1>", self.toggle_auto_image)
```

Comment:
There's nothing wrong with this, but if we only plan to use `toggle_auto_image()` in this one place, we can probably just make it a lambda (assumes code is using properties):
```python
statusbar.add_binding("see img", "<Double-Button-1>", lambda: self.auto_image = not self.auto_image)
```

Reply:
@cpeel , apparently you're not allowed to make assignments in a lambda, so I can't do exactly that. Tempted to leave the toggle function for now, since it's also possible we'll make a decision that everything available in the status bar should also be available in the menus, so the toggle function would appear twice. Open to other suggestions though.

Reply:
TIL! I'd leave it as a function. 👍